webpack/assets/javascripts/react_app/components/notifications/index.js

```diff
@@ -43,7 +43,7 @@ class notificationContainer extends React.Component {
     } = this.props;
 
     return (
-      <div id="notifications_container">
+      <div>
```

Comment:
Removed the `id` field because this component getting mounted into a div with the same `id`.